components/third-party/ingress-nginx/chart/helmRelease.yaml

```diff
@@ -26,6 +26,7 @@ spec:
         chroot: true
       podLabels:
         app: ingress-nginx
+        grafana.radix.equinor.com/monitor-health: "true"
```

Comment:
Should we also configure defaultBackend.podLabels, ref https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx?modal=values&path=defaultBackend.podLabels